<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Attack</class>
 <widget class="QWidget" name="Attack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QLCDNumber" name="m_LCD_HP_2"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="m_Lab_YourCharacter_3">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ваши персонажи</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLCDNumber" name="m_LCD_Damage_3"/>
   </item>
   <item row="3" column="2">
    <widget class="QLCDNumber" name="m_LCD_HP_3"/>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="m_TB_Attack_1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/resource/character/Katana.png</normaloff>:/resource/character/Katana.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="m_TB_Attack_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/resource/character/Katana.png</normaloff>:/resource/character/Katana.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ХП</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="m_Lab_YourCharacter_1">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="m_TB_Attack_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/resource/character/Katana.png</normaloff>:/resource/character/Katana.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="m_Lab_YourCharacter_2">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLCDNumber" name="m_LCD_Damage_1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLCDNumber" name="m_LCD_HP_1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLCDNumber" name="m_LCD_Damage_2"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Урон</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
